modules/swagger-codegen/src/main/resources/LoopBackSwift/model.mustache

```diff
@@ -0,0 +1,46 @@
+{{#models}}{{#model}}import ObjectMapper
+{{#useRealm}}
+import Realm
+import RealmSwift
+{{/useRealm}}
+
+public class {{classname}} : {{#useRealm}}Object, {{/useRealm}}Mappable, CustomStringConvertible {
```

Comment:
As for the examples, there are sample inputs in the project against which we can run the codegen and see what the generated code look like. Or, would you prefer having inline sample codes in comments?